cmd/parca-agent/main.go

```diff
@@ -356,11 +371,11 @@ func run(logger log.Logger, reg *prometheus.Registry, flags flags) error {
 				log.With(logger, "component", "debuginfo"),
 				reg,
 				debuginfoClient,
-				flags.DebuginfoUploadTimeoutDuration,
-				flags.DebuginfoUploadCacheDuration,
-				flags.DebuginfoDirectories,
-				flags.DebuginfoStrip,
-				flags.DebuginfoTempDir,
+				flags.Debuginfo.UploadTimeoutDuration,
+				flags.Debuginfo.UploadCacheDuration,
+				flags.Debuginfo.Directories,
+				flags.Debuginfo.Strip,
+				flags.Debuginfo.CacheDir,
```

Comment:
```suggestion
				flags.Debuginfo.TempDir,
```
This is semantically more correct. We don't cache.